Library/Input_Configuration/OPTIONS_GeneralConfiguration.m

```diff
@@ -278,9 +262,21 @@
                 OPTIONS.ITOL = 0;
         end
     else
-        str = [ 'Error: User selected ITOL: ', num2str(ITOL), '. ITOL must be 0 or 1.' ];
+        str = [ 'Error: User selected ITOL: ', num2str(OPTIONS.ITOL), '. ITOL must be 0 or 1.' ];
         error(str);
-    end    
+    end
+    
+    % MaxOrder
+    if ( OPTIONS.MaxOrder == 0 )
```

Comment:
Similar issue here. Default should be [], and we don't want to silently change bad user inputs.